### DISCUSSION TITLE
Airflow 1.10.15 in production environment

### DISCUSSION THREAD

**Discussion by phaninalluridd** [apache/airflow#27019] [Q&A]
I would like to know if anyone out there running Airflow 1.10.15 in a production environment and any known issues you have encountered.

Context - we are planning our Airflow 2.x upgrade and we may not get there in one scoop as we have to first upgrade to a bridge version, i.e. 1.10.15 and want to know if this bridge version is a stable version to run in production for about 1 to 6 months.

**Comment by potiuk** [comment]
People used it.  Some for longer than that. But you will never get an authoritative answer for your case that you can rely on. 

Why do you need 1 to 6 months ? It makes little sense actually.

I think you should migrate ASAP, there is a growing number of security issues that are not going to be fixed in a product that is end-of-life for 1.5 year now And if there will be a serious one revealed in the future you might have days or hours to migrate not months. 

If you need to convince your management that it needs focus, resources and money to convert, this should be your ulltimate argument - you should ask them whether they are willing to take that risk or not.

It's up to you of course, but to be perfectly honest - asking question here about "whether  it is stable enough" does not help you in anything (other than allowing you to assess the risk). No answer here will help with anything else by lowering some risks in decisions made by you and your management. And the risk of running a software that is known to have vulnerabilities (fixed in Airflow 2 - look at the CVE databases and look for Airflow) and knowing that if anything else will be revealed, it will not be fixed in 1.10 no matter the criticiality is I think enough risk to make you loose a sleep (I would loose a sleep actually if I was responsible for it).

But it's all up to your own risk assesment of course.
